apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -421,32 +422,40 @@ export default function Success(props: PageProps) {
                           "mx-auto flex h-12 w-12 items-center justify-center rounded-full",
                           isRoundRobin &&
                             "border-cal-bg dark:border-cal-bg-muted absolute bottom-0 right-0 z-10 h-12 w-12 border-8",
-                          !giphyImage && !isCancelled && !needsConfirmation ? "bg-success" : "",
-                          !giphyImage && !isCancelled && needsConfirmation ? "bg-subtle" : "",
-                          isCancelled ? "bg-error" : ""
+                          !giphyImage && !isCancelled && !isBookingInPast && !needsConfirmation
+                            ? "bg-success"
+                            : "",
+                          !giphyImage && !isCancelled && !isBookingInPast && needsConfirmation
+                            ? "bg-subtle"
+                            : "",
+                          isCancelled || isBookingInPast ? "bg-error" : ""
                         )}>
-                        {!giphyImage && !needsConfirmation && !isCancelled && (
+                        {!giphyImage && !needsConfirmation && !isCancelled && !isBookingInPast && (
```

Comment:
Can we extract this condition and above two(which are same) in a variable

Reply:
Infact we could make the entire change simpler by having a new variable say `isReschedulable=!isCancelled && !isBookingInPast `

and then we could just use `isReschedulable` in place of `!isCancelled`